packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1049,6 +1049,47 @@ class PhysicalModel extends SingleChildRenderObjectWidget {
 /// [PhysicalModel] does the same but only supports shapes that can be expressed
 /// as rectangles with rounded corners.
 ///
+/// {@tool dartpad --template=stateless_widget_material}
+///
+/// This example shows how to use a [PhysicalShape] on a centered [SizedBox]
+/// to clip it to a rounded rectangle using a [ShapeBorderClipper] and give it
+/// an orange color along with a shadow.
+///
+/// ```dart
+/// Widget build(BuildContext context) {
+///   return Scaffold(
+///     appBar: AppBar(
+///       title: const Text('PhysicalShape Sample'),
+///     ),
+///     body: Center(
+///       child: PhysicalShape(
+///         elevation: 5.0,
+///         child: const SizedBox(
+///           child: Center(
+///             child: Text(
+///               'Hello, World!',
+///               style: TextStyle(
+///                 color: Colors.white,
+///                 fontSize: 20.0
```

Comment:
nit: add a trailing comma

```suggestion
///                 fontSize: 20.0,
```